### PR TITLE
Update geometry.rst

### PR DESCRIPTION
**Pull request by a-lambda**
Goal: propose an answer to [16.5.6. Try Yourself](https://docs.qgis.org/3.34/en/docs/training_manual/spatial_databases/geometry.html#id1)

*Create city boundaries by computing the minimum convex hull of all addresses for that city and computing a buffer around that area.*

I don't know if you will be ok with that.
Regards

- [ ] Backport to LTR documentation is requested

### CODE REVIEW

**Comment by a-lambda** [comment]
I pulled the request too fast.
I must fix some indentations and formatting errors.

File: docs/training_manual/spatial_databases/geometry.rst [hunk 1]
```diff
@@ -194,6 +194,67 @@ Try Yourself: :abbr:`★★★ (Advanced level)`
...
+   .. code-block:: psql
+
+    CREATE TABLE myPolygonTable (
+      id serial NOT NULL PRIMARY KEY,
+      city_id int NOT NULL REFERENCES cities(id),
+      geometry geometry NOT NULL
+    );
+
+    ALTER TABLE myPolygonTable
+      ADD CONSTRAINT myPolygonTable_geom_point_chk
```
Comment (DelazJ) [inline docs/training_manual/spatial_databases/geometry.rst:239]:
```suggestion
      ADD CONSTRAINT myPolygonTable_geom_polygon_chk
```

**Review by DelazJ — COMMENTED** [review]
@a-lambda of course it is a good idea; a"try yourself" should have an answer. I have a few suggestions about geometry field naming. And not sure adding a buffer is correct...

File: docs/training_manual/spatial_databases/geometry.rst [hunk 2]
```diff
@@ -194,6 +194,62 @@ Try Yourself: :abbr:`★★★ (Advanced level)`
...
 for that city and computing a buffer around that area.
 
+.. admonition:: Answer
+  :class: dropdown
+
+   .. code-block:: psql
+
+   --* Add some people in 'Tokyo Outer Wards' city
+
+    INSERT INTO people (name, house_no, street_id, phone_no, city_id, the_geom)
```
[the remark this answers is not in the document]
Comment (a-lambda) [reply docs/training_manual/spatial_databases/geometry.rst]: changes committed

File: docs/training_manual/spatial_databases/geometry.rst [hunk 3]
```diff
@@ -194,6 +194,67 @@ Try Yourself: :abbr:`★★★ (Advanced level)`
...
+               'SRID=4326;POINT(29 18)');
+
+  - Create myPolygonTable table  
+
+   .. code-block:: psql
+
+    CREATE TABLE myPolygonTable (
+      id serial NOT NULL PRIMARY KEY,
+      city_id int NOT NULL REFERENCES cities(id),
+      geometry geometry NOT NULL
```
[the remark this answers is not in the document]
Comment (a-lambda) [reply docs/training_manual/spatial_databases/geometry.rst]: changes committed

**Comment by a-lambda** [comment]
Doctest build always fail.
I don't know what to do for that.
Regards.

**Comment by DelazJ** [comment]
Thanks @a-lambda 
> Doctest build always fail.

Not your fault. I'm on it.

**Comment by a-lambda** [comment]
Thanks for Help.
Regards

**Comment by DelazJ** [comment]
Direct backport to 3.40 in 6179782

**Pull request merged**